tests/test_admin.py

```diff
@@ -15,4 +15,6 @@ def test_create_admin():
             check.append(user.has_role('ADMIN'))
         except AttributeError:
             assert False, 'Admin accounts not created'
+    if len(check) == 0:

```

Comment:
More pythonic way to check if a list is empty is to do
```
a = []
if not a:
    print("Empty list")
else:
    print("Non-empty list")
```
https://stackoverflow.com/a/53752